modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.js

```diff
@@ -104,6 +104,44 @@ const TooltipProvider = () => {
 		return dispose;
 	}, [delay, state]);
 
+	const getContent = element => {
+		if (element) {
+			const title = element.getAttribute('title');
+
+			if (title) {
+				element.setAttribute('data-title', title);
```

Comment:
We should not set `data-title`, otherwise now we'll have

```html
<div title="foo">...</div>
<div data-title="bar">...</div>
```

turn into

```
<div data-title="foo" title="foo">...</div>
<div data-title="bar">...</div>
```

Just set the value in `data-restore-title` and restore that instead of storing a boolean and using the other attribute for the value